src/ui.c

```diff
@@ -64,7 +64,7 @@ file_finder_read(struct file_finder *finder, const char *commit)
 			break;
 		}
 
-		finder->file[files] = calloc(1, sizeof(*finder->file) + buf.size);
+		finder->file[files] = calloc(1, sizeof(*finder->file) + buf.size + 1);
```

Comment:
Did you see this default? The `text` field in `file_finder_line` has size 1 to account for the NUL byte.